packages/zoe/src/zoeService/zoe.js

```diff
@@ -244,27 +244,23 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
           getIssuers: () => instanceRecord.terms.issuers,
           getBrands: () => instanceRecord.terms.brands,
           getInstance: () => instance,
-          hasShutdown: () => hasShutdown,
-          shutdown: () => {
-            hasShutdown = true;
+          acceptingOffers: () => acceptingOffers,
+          exitAllSeats: () => {
+            acceptingOffers = false;
             zoeSeatAdmins.forEach(zoeSeatAdmin => zoeSeatAdmin.exit());
           },
```

Comment:
```suggestion
          exitAllSeats: completion => {
            acceptingOffers = false;
            zoeSeatAdmins.forEach(zoeSeatAdmin => zoeSeatAdmin.exit(completion));
          },
```

Reply:
This isn't a kickOut, so I think the passing of a completion message isn't right

Reply:
`kickOut` should take a `reason`, not a `completion`.

`exit` and therefore `exitAllSeats` is to `kickOut` as `terminate` is to `terminateWithFailure`. Perhaps `kickOut` should be renamed `exitWithFailure` to emphasize the analogy. The seat completion should be used at the seat's notifier's completion. See above.